readme.md

```diff
@@ -119,15 +151,20 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> the bang (!) modifies the @artist object its called on.
 
```

Comment:
The difference between `.save` and `.save!` is that the latter will generate an error and halt the application if the artist is not saved properly. The former will silently error and stop the application.